lib/rspec/support/object_formatter.rb

```diff
@@ -3,9 +3,23 @@ module Support
     # Provide additional output details beyond what `inspect` provides when
     # printing Time, DateTime, or BigDecimal
     module ObjectFormatter
+      class << self
+        attr_accessor :max_formatted_output_length
+      end
```

Comment:
> Should the max_formatted_ouput_length have a default value?

Yes.

> If so, what should it be?

Off the top of my head, 200 characters sounds about right, but I'd have to see comparisons of output at different levels to say for sure.  Can we go with 200 for now, and then maybe after you have it working, you can paste snippets  or screenshots of example output at different length values?

> Where should the shared instance be defined?

Here is fine.  Something like:

``` ruby
@default_instance = new

def self.format(object)
  @default_instance.format(object)
end
```

> I've used Forwardable for delegation but not to an instance - would it be possible to use Forwardable for that?

I believe it would be possible, but we have a policy of using as little of the stdlib as possible.  If we used forwardable here, that would allow a gem author to write a gem that uses forwardable and forget to require it.  There RSpec test suite would still be green (since RSpec would load forwardable due to your use of it here) but their gem would be broken -- users of their gem that were not already requiring forwardable would get errors.  We try to avoid these situations by minimizing our use of the standard library.